src/app.ts

```diff
@@ -0,0 +1,93 @@
+import express, { Application, Request, Response, NextFunction } from "express";
+import * as bodyParser from "body-parser";
+import { check, validationResult } from "express-validator";
+
+const app: Application = express();
+
+app.use(bodyParser.json());
+
+interface todo {
```

Comment:
look like it's a `type` not `interface`

Reply:
fixed on 5e1eb54c01ef4a6b08ebc2d1c553572e5fa17e1a